<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Open local playlist</string>
  </property>
  <widget class="QLabel" name="lbl_url">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>25</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_url">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>21</y>
     <width>631</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>c:\some\path\playlist.m3u</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_save">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Remember</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_browse">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_preview">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>591</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_preview">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Preview names</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_pl_sep_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>205</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_pl_sep_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Category separator</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_pl_sep_1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>205</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_pl_sep_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Group separator</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
